src/encoded/auth0.py

```diff
@@ -96,6 +101,56 @@ def forget(self, request):
         return []
 
 
+@view_config(context=User.Collection, request_method='POST', permission=NO_PERMISSION_REQUIRED, name='sign-up')
```

Comment:
You should make a new permission (maybe called `signup`) and add it to this list of permissions that are assigned to everyone: https://github.com/ENCODE-DCC/encoded/blob/eea4bf93e65f8368a54b16acb098e6daea1cfa2a/src/encoded/root.py#L56

Then make this view use the `signup` permission instead of NO_PERMISSION_REQUIRED. This is similar to how the search/report/matrix views have a `search` permission.